llvm/test/CodeGen/NVPTX/rcp-opt.ll

```diff
@@ -0,0 +1,34 @@
+; RUN: llc < %s -march=nvptx64 | FileCheck %s
```

Comment:
This test will benefit from autogenerated checks:
https://llvm.org/docs/TestingGuide.html#generating-assertions-in-regression-tests

Reply:
Fixed in the latest revision.